Makefile

```diff
@@ -0,0 +1,9 @@
+PYTHON ?= python
+
+all: linter
+
+cpplint.py:
+	curl -o cpplint.py "https://raw.githubusercontent.com/nodejs/node/master/tools/cpplint.py"
+
+linter: cpplint.py
```

Comment:
In nodejs/node, the target is called `cpplint` and there's an overarching `lint` target for linting C++ and JS code; you may want to follow that example here.

Can you add a `.PHONY: all cpplint lint` rule?